chart/values.yaml

```diff
@@ -0,0 +1,123 @@
+# Default values for k8s-janitor.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+managedByHelm: true
+environment: "develop"
+
+image:
+  repository: 579478677147.dkr.ecr.eu-central-1.amazonaws.com/ded/k8s-janitor
+  pullPolicy: Always
+  tag: "latest"
```

Comment:
The previous deployment used a specific build, should we also implement the functionality to do so here, so that it will be easier to switch from using the old deployment for production to this one?